src/EventInvocation.js

```diff
@@ -239,7 +239,7 @@ export class EventInvocation {
 
   // This method will clear the eventListeners and the event hsitory for the listener as a part of FCA
   clearAllListeners() {
-    logger.info('Going to clear  registering listeners' + eventHandlerMap, 'clearAllListeners');
+    logger.info('Going to clear registering listeners' + JSON.stringify(eventHandlerMap), 'clearAllListeners');
```

Comment:
Rephrase to `Clearing registered listeners`

Reply:
Addressed the comment